x-pack/plugins/security_solution/public/detections/components/value_lists_management_modal/flyout.tsx

```diff
@@ -67,7 +66,7 @@ export const ValueListsModalComponent: React.FC<ValueListsModalProps> = ({
   const [exportDownload, setExportDownload] = useState<{ name?: string; blob?: Blob }>({});
   const { addError, addSuccess } = useAppToasts();
   const [showReferenceErrorModal, setShowReferenceErrorModal] = useState<boolean>(false);
-  const [referenceModalState, setReferenceModalState] = useState<ReferenceModalState>(
+  const [referenceFlyoutState, setReferenceFlyoutState] = useState<ReferenceFlyoutState>(
     referenceModalInitialState
```

Comment:
```suggestion
  const [referenceFlyoutState, setReferenceFlyoutState] = useState<ReferenceFlyoutState>(
    referenceFlyoutInitialState
```

Reply:
This one (the reference modal) is still a modal that pops out so gonna leave that one.